lib/Driver/Driver.cpp

```diff
@@ -221,8 +221,12 @@ makeToolChain(Driver &driver, const llvm::Triple &target) {
   case llvm::Triple::FreeBSD:
     return llvm::make_unique<toolchains::GenericUnix>(driver, target);
     break;
-  case llvm::Triple::Win32:
-    return llvm::make_unique<toolchains::Cygwin>(driver, target);
+  case llvm::Triple::Win32: 
+    if (target.isWindowsCygwinEnvironment()) {
+      return llvm::make_unique<toolchains::Cygwin>(driver, target);
+    } else {
+      return llvm::make_unique<toolchains::Windows>(driver, target);
+    }
```

Comment:
Unnecessary braces.

Reply:
This matches the `target.isAndroid` conditional above. What's the Swift project style on this? Should I change both of them to remove the braces?

Reply:
We're based on LLVM style, which says no braces for single-line if-statements. We swing pretty heavily on the side of having them whenever it's a grey area, though:

- If you have comments inside the body, provide the braces.
- If the line wraps, people usually provide the braces.
- If another branch of the `if` has braces, people usually add braces to all of them.
- If the *condition* wraps, sometimes people provide braces.